src/librustdoc/html/static/js/scrape-examples.js

```diff
@@ -106,10 +120,18 @@
         }
     };
 
+    /**
+     * Intitialize the `locs` field
+     *
+     * @param {HTMLElement & {locs?: rustdoc.ScrapedLoc[]}} example
+     * @param {boolean} isHidden
+     */
     function setupLoc(example, isHidden) {
-        example.locs = JSON.parse(example.attributes.getNamedItem("data-locs").textContent);
+        const locs_str = example.attributes.getNamedItem("data-locs").textContent;
+        const locs = example.locs =
+              JSON.parse(nonnull(nonnull(locs_str));
```

Comment:
```suggestion
              JSON.parse(nonnull(nonnull(locs_str)));
```

Unbalanced parentheses (syntax error).

Reply:
huh. weird that tsc didn't catch that.

Reply:
oh that's wacky, it didn't catch that because that's not what's in my local repo, lol.

wonder how that happened.

Reply:
Are you referring to your locally installed `tsc` (if any) which you ran locally on your machine or are you referring to the one run in CI?

The CI `tsc` runs *after* `es-check`. And it's the *latter* which gave you the CI error "unexpected token at line 132" and which presumably made the job exit early. The error message provided by the Acorn JS parser isn't great but it's at least something ^^'.

Reply:
local tsc. ultimately the problem is the CI was running against different code than I had locally. probably I messed up the git push command so it actually pushed nothing.  not a failure mode i think i've expirenced more, but it was probably related to me having to do fancy stuff since i accidentally touched submodules.